hw/ip/prim/dv/prim_lfsr/data/prim_lfsr_cover.cfg

```diff
@@ -0,0 +1,9 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifer: Apache-2.0
+
++tree prim_lfsr_tb.gen_duts[24].i_prim_lfsr
```

Comment:
nit: perhaps good to copy the commit message bullet 3 here to explain the rational

Reply:
good idea, will do so.